specification/src/main/asciidoc/chapters/Sparkplug_4_Topics.adoc

```diff
@@ -151,47 +148,104 @@ infrastructure.
 values.
 * Issue write/command messages to any Edge Node or Device metric.
 
-This section of the document defines the topic namespace and how each of the associated messages
-types can be used.
+This section defines the payload contents and how each of the associated messages types can be used.
 
 [[topics_edge_node]]
 ==== Edge Node
 [upperalpha, start=1]
 
-[[topics_birth_message_nbirth]]
+[[birth_message_nbirth]]
 ===== Birth Message (NBIRTH)
 
+[[topics_birth_message_nbirth]]
+====== Topic (NBIRTH)
+
 The Birth Certificate topic for an Sparkplug Edge Node is:
 [subs="quotes"]
   namespace/group_id/*NBIRTH*/edge_node_id
 
+[[payloads_desc_nbirth]]
+====== Payload (NBIRTH)
+
 The Sparkplug Edge Node Birth Certificate payload contains everything required to build out a data
 structure for all metrics for this Edge Node. At the time any host application receives and NBIRTH,
 the ONLINE state of this Edge Node should be set to TRUE along with the associated ONLINE Date Time
 parameter. Note that the Edge Node Birth Certificate ONLY indicates the node itself is online and in
 an MQTT Session, but any devices that have previously published a DBIRTH will still have “*STALE*”
 metric quality until the host application receives the associated DBIRTH messages.
 
-[[topics_data_message_ndata]]
+The NBIRTH message requires the following payload components.
+
+* The NBIRTH must include the a seq number in the payload and it must have a value of 0.
+* The NBIRTH must include a timestamp denoting the DateTime the message was sent from the EoN node.
+* The NBIRTH must include every metric the EoN node will ever report on. At a minimum these metrics must 
+include:
+** The metric name
+** The metric datatype
+** The current value
+* If Template instances will be published by this EoN or any devices, all Template definitions must be 
+published in the NBIRTH.
+* A bdSeq number as a metric should be included in the payload. This should match the bdSeq number provided 
```

Comment:
I think a bdSeq number MUST be included.

Reply:
Ok.  I hadn't changed that text.  Will update